x-pack/plugins/apm/public/components/shared/charts/timeseries_chart.tsx

```diff
@@ -219,7 +223,7 @@ export function TimeseriesChart({
           tickFormat={yTickFormat ? yTickFormat : yLabelFormat}
           labelFormat={yLabelFormat}
         />
-
+        {rectsAndAnnotations}
```

Comment:
I'm not a fan of us having both `annotations` / `showAnnotations` and `rectAnnotations`. is there another way?

Reply:
@dgieselaar, I see your point. Maybe instead of going generic, I could be more specific with something like `alertDetailsChartExtraInfo`
`alertDetailsChartAdditionalData`

Reply:
I like `alertDetailsChartAdditionalData` more, @dgieselaar what do you think?

Reply:
updated